### PR TITLE
Adding konvoyconfig

### PR DESCRIPTION
**Pull request by jr0d**

### CODE REVIEW

File: templates/konvoyconfig.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+      enabled: true
+    - name: none
+      enabled: true
+  chartReference:
+    chart: konvoyconfig
+    repo: https://mesosphere.github.io/charts/staging
+    version: 0.0.2
+    values: |
+      ---
+      clusterHostname: ""
```
Comment (shaneutt) [inline templates/konvoyconfig.yaml]: :thinking: would we want to add a small comment here explaining to the reader the full context of how and why this configuration option works like this?

**Review by shaneutt — APPROVED** [review]
One non-blocking comment (but still might be worth updating for).

Marking this as blocked on https://github.com/mesosphere/charts/pull/141 as that needs to merge before this can.

**Comment by hectorj2f** [comment]
Yeah, let's wait until the PR is merged

File: templates/konvoyconfig.yaml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    - name: aws
+      enabled: true
+    - name: docker
+      enabled: true
+    - name: none
+      enabled: true
+  chartReference:
+    chart: konvoyconfig
+    repo: https://mesosphere.github.io/charts/staging
+    version: 0.0.2
```
Comment (hectorj2f) [inline templates/konvoyconfig.yaml]: This version is 0.0.1

**Review by hectorj2f — CHANGES_REQUESTED** [review]

**Comment by jr0d** [comment]
@hectorj2f fixed

**Review by hectorj2f — APPROVED** [review]

**Pull request merged**